Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+**Oleh Kulykovskyi** - front-end developer with 4.5 years of work experience including 3.5 years working with React & Next.JS, mostly in TypeScript language. Proficient with numerous business-grade tools like Redux, GraphQL, Git, Github, animation libraries (GSAP, Framer), MUI, Jest etc. Implemented a broad range of web solutions from landing pages focused on visuals to complex system of business-grade interconnected apps. Created and implemented a web3 platform for a project in Polkadot ecosystem - [Asylum space](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md). Developed an npm package as a solution for integrating blockchain into React. Tech stack: JS, React.js, Next.js, web3.js, polkadot.js, Typescript.
+
+**Artem Smolyarenko** - UI/UX designer with over 3 years experience. He specializes in brand development in all aspects and have an excellent understanding of typography, color psychology, as well as grid, web and mobile principles. Artem worked on different types of products with startups and corporate teams from around the world. Skills: UI/UX design, product design, web and mobile app design. Tech stack & tools: Figma, Adobe Photoshop, Adobe Illustrator, Adobe XD.
+
+### Team Code Repos
+
+- **Archisinal** [https://github.com/Archisinal](https://github.com/Archisinal)
+- **Danylo Semirazov** - [github.com/sventime](https://github.com/sventime), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+- **Oleh Kulykovskyi**  - [github.com/blindbeat](http://github.com/blindbeat), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+
+### Team LinkedIn Profiles (if available)
+
+- **Tom Humber** https://www.linkedin.com/in/tom-humber-b65888118
+- **Danylo Semirazov** - [linkedin.com/in/danylosemirazov/](https://www.linkedin.com/in/danylosemirazov/)
+- **Oleh KulyKovskyi**  - [linkedin.com/in/oleh-kulykovskyi-329523234/](http://www.linkedin.com/in/oleh-kulykovskyi-329523234/)
+
+
+## Development Status :open_book:
+
+- [Medium](https://medium.com/@archisinal)
+- [Twitter](https://twitter.com/archisinal)
+- [Instagram](https://www.instagram.com/archisinal.io/)
+
+## Development Roadmap :nut_and_bolt: ##
+
+### Overview
+
+The development roadmap for Archisinal includes multiple milestones to ensure the successful launch and operation of the platform.
+
+### **Overview**
+
+- **Total Estimated Duration:**
+    - MVP: 2.5 months
+    - Post-MVP: 9 months
+    - Post-Launch: 10 months
+- **Full-Time Equivalent (FTE): 6**
+- **Total Costs:**
+    - MVP: $60,000
+
+## MVP (brief discussion)
+
+### **Milestone 1: Smart Contract Development**
```

Comment:
I'm not opposed to having multiple implementation of the same thing, especially in an emerging ecosystem - but why are you choosing to write your own contracts instead of building on top of something like RMRK or ArtZero?

Reply:
Thank you for your valuable comment. I have provided my arguments about RMRK in the [comment above](https://github.com/use-inkubator/Ecosystem-Grants/pull/19#discussion_r1196634848).

And would also like to admit that [ArtZero Contracts](https://github.com/ArtZero-io/Contracts) aligns more closely with our use case, as it includes marketplace functionality, auction features, and the ability to post and edit projects. However, some of these features exceed our requirements. In Archisinal, particularly during the MVP stage, we intend to maintain a very simple flow where each NFT project is personalized for a specific client and managed by the creator of the NFT.

Considering the points mentioned above, along with the comment from @Archisinal regarding the future plans of the platform, integrating with RMRK or ArtZero at this stage, especially for the MVP, could introduce more difficulties than benefits.

Reply:
Circling back to this @HCastano - I believe the RMRK integration is no longer feasible as they focus more on EVM solutions. From my point of view, this reinforces why we need to build out our own solution, as to not have a reliance on another team to support our tech stack.